doc/nrf/ug_bt_coex.rst

```diff
@@ -49,15 +49,15 @@ Run-time configuration
 Per-state priorities
```

Comment:
Commenting here since I can't comment on a commit message:
It seems you have an extra commit "Bluetooth: coex: add init on boot option in Kconfig" is the commit message but it doesn't contain anything related to that, only renaming the commands.
I think you should put all the changes in this PR in the previous commit

Reply:
True, it squash it